src/Microsoft.DotNet.Interactive.Formatting.Tests/TabularDataFormatterTests.can_generate_tabular_json_from_object_array.approved.json

```diff
@@ -20,27 +20,27 @@
     {
       "Name": "Q",
       "IsValid": false,
-      "Cost": 10.0
+      "Cost": 10
```

Comment:
This is contract change that I would expect might impact the integration with the data explorer, no?

Reply:
to ensure number serialization preserves decimals, the new serialization seems to be too smart about decimals with 0 value after the period